packages/framework/composer.json

```diff
@@ -83,6 +69,8 @@
         "psr/log": "^1.0",
         "sensio/distribution-bundle": "^5.0.21",
         "sensio/framework-extra-bundle": "^3.0.29",
+        "shopsys/jparser": "0.1",
+        "shopsys/postgres-search-bundle": "0.1",
```

Comment:
The tag `0.1` was decided during PIA. We can change it, these repositories are not part of monorepo. We want to declare that this is something we just had to fork and released tag because the original package didn't have any. And since we don't understand these packages too much, we think that more than `0.1` like `1.0` would say to the world "this is stable and we declare it", but it is not our case.

No, I didn't consider copying classes into framework.

Reply:
Copying classes from product-search-bundle to framework is in my opinion out of the scope of this US